tests/test_xdist.py

```diff
@@ -83,3 +83,38 @@ def test_a():
     assert result.stderr.lines == snapshot([])
 
     assert result.ret == 1
+
+
+def test_xdist_zero_processes(project):
+
+    project.setup(
+        """\
+
+def test_a():
+    assert 1==snapshot(2)
+"""
+    )
+
+    result = project.run("-n=0")
```

Comment:
```suggestion
    result = project.run("-n=0","--inline-snapshot=fix")
```
This should fix the ci problem.

Reply:
Thanks, I went with short-report instead to shorten the test.